main/gps_backgroundtasks.go

```diff
@@ -0,0 +1,686 @@
+/*
```

Comment:
These are teh background tasks that where in gps.go but now have it's seperate place